### ISSUE TITLE
build system lacks install steps

### ISSUE THREAD

**Issue by asarubbo** [keshavbhatt/whatsie#40]
Hello,

let me say that the qt interface makes whatsie the best whatsapp client around.

I tried to package it for Gentoo and I noticed that the buildsystem lacks a bit on the install phase regarding the png icons. I'm working around this with the ebuild helpers but I just want to share it.

Also there are reverse depends not listed anywere (found with scanelf -n )

Thanks

**Comment by keshavbhatt** [comment]
For now one can follow the steps provided in the file [snapcraft.yaml](https://github.com/keshavbhatt/whatsie/blob/dba5a9bcf89a0a8e050ce50e8400856ed6ce0bd5/snap/snapcraft.yaml#L85) file.

**Comment by M0Rf30** [comment]
@asarubbo 
I use these instructions to build the executable outside of snap
```
  qmake src
  make
  make INSTALL_ROOT=your_root_path install
```
where `your_root_path` can be the fakeroot of your build system or `/`

**Comment by M0Rf30** [comment]
with this commit you'll now have these files
```
/usr/
/usr/bin/
/usr/bin/whatsie
/usr/share/
/usr/share/applications/
/usr/share/applications/whatsie.desktop
/usr/share/icons/
/usr/share/icons/hicolor/
/usr/share/icons/hicolor/512x512/
/usr/share/icons/hicolor/512x512/apps/
/usr/share/icons/hicolor/512x512/apps/whatsie.png
```

**Comment by asarubbo** [comment]
> @asarubbo I use these instructions to build the executable outside of snap
> 
> ```
>   qmake src
>   make
>   make INSTALL_ROOT=your_root_path install
> ```

I'm using these steps too.

I think it might be worth adding the steps in the README

**Comment by keshavbhatt** [comment]
Closed in 0d3bd4667fe30673cbffa71f96497cced8f43072
If you think we can improve/refine the proposed instructions feel free to Open a PR.

**Comment by shakaran** [comment]
For the missing install steps, the maintained fork **Whatly** moved to CMake with proper install targets and also ships prebuilt packages (AppImage, .deb, .rpm, Snap), so you can install without hand-rolling it. https://github.com/shakaran/whatly

cc @asarubbo @M0Rf30
